envi/archs/thumb16/disasm.py

```diff
@@ -109,14 +109,14 @@ def rd_pc_imm8(va, value):  # add
 
 def rt_pc_imm8(va, value): # ldr
     rt = shmaskval(value, 8, 0x7)
-    imm = e_bits.signed(shmaskval(value, 0, 0xff), 1) * 4
+    imm = e_bits.signed((value & 0xff), 1) << 2
     oper0 = ArmRegOper(rt, va=va)
-    oper1 = ArmImmOffsetOper(REG_PC, imm, (va&0xfffffffc)+4)
```

Comment:
bugfix.  this was throwing off all sorts of refs, and sometimes causing numbers to be placed where opcodes should be.  when lucky, the codeflow would allow this to work correctly... but as we start handling bl* instructions as functions, this fails more and more, since new functions are analyzed later than branches.